sdk/src/client/bindings/python/iota_client/types/unlock_condition.py

```diff
@@ -0,0 +1,142 @@
+from enum import Enum
+
+
+class UnlockConditionType(Enum):
+    Address = 0
+    StorageDepositReturn = 1
+    Timelock = 2
+    Expiration = 3
+    StateControllerAddress = 4
+    GovernorAddress = 5
+    ImmutableAliasAddress = 6
+
+
+class UnlockCondition():
+    def __init__(self, type=None, address=None, amount=None, unix_time=None, return_address=None) -> None:
```

Comment:
What's that `-> None:` for ?

Reply:
If I understand https://stackoverflow.com/a/38286728 correctly it just means that this method returns None and can safely be removed

Reply:
Removed